packages/vscode-mdx/package.json

```
@@ -66,7 +66,7 @@
         "properties": {
           "mdx.experimentalLanguageServer": {
```

I would like to rename this property to mark this as no longer experimental. I’m currently thinking of `mdx.server.enable`, but I’m open to suggestions. (`<feature>.enable` is a quite common unofficial convention for VSCode settings.)

```suggestion
          "mdx.server.enable": {
```

---

LGTM!